.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
     hooks:
       # Run the linter.
       - id: ruff
-        args: [ ., --fix, --diff ]
+        entry: ruff check . --fix
```

Comment:
I removed the `--diff` option because it was difficult to understand the content of the ruff error without displaying it!